D1_datasets.md

```diff
@@ -0,0 +1,61 @@
+# Datasets
+
+## Long Term Experiment
+
+### Overview
+
+This dataset offers a compilation of long-term dynamic motion data gathered over approximately 3.5 weeks from a newly designed 4-DoF tendon-driven robotic arm powered by pneumatic artificial muscles (PAMs).
+The data encompasses movements generated by random multisine signals of target pressures, as well as repeated fixed target pressure movements, facilitating the evaluation of system repeatability and performance.
+
+### Robot
+
+The dataset was taken on Pamy2
+
+### Files
+
+The files can be downloaded on [Edmond](https://edmond.mpdl.mpg.de/dataset.xhtml?persistentId=doi:10.17617/3.OMM0JP).
+There are 6 zip files. Each files contains several "df" files (around 40), each file containning a pandas dataframe.
```

Comment:
Small spelling issues:
- Each file contains (not files)
- each file containing (not containning)